Dockerfile.launcher

```diff
@@ -0,0 +1,12 @@
+# To build it:
+#  `docker build -t codenvy/che-launcher -f Dockerfile.launcher .`
+# To use it:
+#  `docker run --rm \
+#       -v /var/run/docker.sock:/var/run/docker.sock \
+#        codenvy/che-launcher [start|stop|restart|update]
+FROM docker:1.11
```

Comment:
The `$(which docker)` syntax will fail on Docker for Windows.  But you can do a string parse using `docker version`. There is go format syntax accepted as part of the command. But this does seem like an inelegant solution.  All this work to simplify the run syntax to just introduce a complicated mount to handle the docker client binary.

Seems like we need to embed smart logic within the launcher.sh to figure out what docker daemon version is running, test for version compatibility, and then use the right docker client within the launcher.